src/schema/entities.yaml

```diff
@@ -0,0 +1,135 @@
+---
+sub:
+  name: Subject
+  description: |
+    A person or animal participating in the study.
+  format: label
+ses:
+  name: Session
+  description: |
+    A logical grouping of neuroimaging and behavioral data consistent across
+    subjects.
+    Session can (but doesn't have to) be synonymous to a visit in a
+    longitudinal study.
+    In general, subjects will stay in the scanner during one session.
+    However, for example, if a subject has to leave the scanner room and then
+    be re-positioned on the scanner bed, the set of MRI acquisitions will still
+    be considered as a session and match sessions acquired in other subjects.
+    Similarly, in situations where different data types are obtained over
+    several visits (for example fMRI on one day followed by DWI the day after)
+    those can be grouped in one session.
+    Defining multiple sessions is appropriate when several identical or similar
+    data acquisitions are planned and performed on all -or most- subjects,
+    often in the case of some intervention between sessions (e.g., training).
+  format: label
+task:
+  name: Task
+  format: label
+  description: |
+    Each task has a unique label that MUST only consist of letters and/or
+    numbers (other characters, including spaces and underscores, are not
+    allowed).
+    Those labels MUST be consistent across subjects and sessions.
+acq:
+  name: Acquisition
+  description: |
+    The OPTIONAL acq-<label> key/value pair corresponds to a custom label the
```

Comment:
if ``` `TXT` ``` is allowed in yaml, we better make those `acq-<label>` into ``` `acq-<label>` ``` right away (as well as sample filenames which could have some characters which markdown could interpret for formatting) to simplify any possible future rendering...  Since we do not render them currently though -- I think it would be ok to proceed as is (unless some other really needed changes are suggested) and just defray it to a separate PR